### DISCUSSION TITLE
GOSS parameters in task-oriented AutoML

### DISCUSSION THREAD

**Discussion by ohenry-proda** [microsoft/FLAML#863] [Q&A]
Hi,

I'm trying to tune a GOSS-enabled `LGBMRegressor` using the `AutoML` and `LGBMEstimator` from FLAML. This works great but it doesn't seem to tune the `top_rate` and `other_rate` associated with GOSS. Is there a way to enable this without wrapping the estimator in a function and tuning that?

**Comment by sonichi** [comment]
Have you considered the shortcut to override the search space? https://microsoft.github.io/FLAML/docs/Use-Cases/Task-Oriented-AutoML#a-shortcut-to-override-the-search-space
